pages/Search/selectors.js

```diff
@@ -1,14 +1,21 @@
 import { createSelector } from 'reselect';
-import { getProductsResult } from '@shopgate/pwa-common-commerce/product/selectors/product';
-import { hasActiveFilters } from '@shopgate/pwa-common-commerce/filter/selectors';
```

Comment:
I guess If we have a "filtered search" without result the filter bar should be still shown, right?